source/server/parser.cpp

```diff
@@ -18,6 +18,7 @@ tokenList Server::parse(std::string buffer)
     std::string line;
     std::vector<std::string> strList;
     tokenList list;
+    std::string eof("\r\n");
```

Comment:
Tens o EOL(End Of Line) no Server.hpp, não precisas de criar outro.

Reply:
P.S.: Se quiseres muda o nome da constante mas tens que mudar em todo o sítio onde ela é usada. Usei EOL porque EOF está indisponível por já estar definido como macro